routes/dodaj.js

```diff
@@ -20,13 +20,11 @@ const dodaj = (req, res) => {
       }
     }
 
-    console.log("RES", res)
-
     mydb.collection("spomenici").insertOne(model, (err, res) => {
       if (err) throw err
       db.close()
-      res.send(`Dodat Spomenik ${naslov}`)
     })
```

Comment:
Zasto si pomerio ovu liniju? Mislim da bi poruka o uspehu trebalo da bude unutar callbacka.

Reply:
zato sto mi je izbacivala gresku, res.send is not a valid function. Iako se ubaci spomenik, server padne svaki put nakon POST requesta.

Reply:
inace sve rute su testirane u postman-u.

Reply:
mozemo da refaktorisemo da izgleda ovako nesto:
itemsCollection.insertOne(model)
  .then(result => console.log(`Successfully inserted item with _id: ${req.params.id}`))
  .catch(err => console.error(`Failed to insert item: ${err}`))

Reply:
mislim da je problem bio u redosledu, prvo treba da posaljes odgovor, a onda da zatvoris konekciju. probaj, pa ako ne bude radilo sredicemo uzivo.